hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/S3GatewayConfigKeys.java

```diff
@@ -81,6 +81,9 @@ public final class S3GatewayConfigKeys {
   public static final boolean OZONE_S3G_LIST_KEYS_SHALLOW_ENABLED_DEFAULT =
       true;
 
+  public static final String OZONE_S3G_METRICS_PERCENTILES_INTERVALS_SECONDS_KEY
+      = "ozone.s3g.metrics.percentiles.intervals.seconds";
```

Comment:
I don't see a default value being specified.

Reply:
Have Added to `addPropertiesNotInXml`

Reply:
I am not sure how adding it to `addPropertiesNotInXml` in integration tests will help customers trying to deploy and use this metric? It makes sense to have some defaults specified?

Reply:
This configuration is used to control the time range of the data set, when exceeding this time, the previous data will be cleaned up, and then re-statistics
So the most suitable value for this configuration is the same as the time interval of the user to collect the indicator. 
For example, if the user collects the indicator every 30s, then configure the indicator to 30s, so that the user will get new data every 30s, and no data will be lost. But if the user collects a metric every 60s, then half of the metrics will be discarded and not captured.
If want to set a default value, perhaps set a larger value, such as 60s or 120s, so that as long as the user collects at an interval less than the default value, at least the indicator won't be missed (though it's possible that some of the indicators may be close in value).

Reply:
> It makes sense to have some defaults specified

Thank you, @xichen01. +1 on specifying a default value and adding the same to `ozone-default.xml`!

Reply:
Done. Set default value to 60s.